projects/vdk-plugins/vdk-jupyter/build-job-tutorial-temp/create_tutorial.md

```diff
@@ -0,0 +1,62 @@
+# Learn How to Create  a Data Job through the Jupyter UI
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+
+Before you start with the build, please ensure you have the following:
+
+<details>
+  <summary><b>Installed Software and Services</b></summary>
+
+- **VDK:** Ensure that the Versatile Data Kit is installed and properly configured.
+- **[Control Service](https://github.com/vmware/versatile-data-kit/wiki/Interfaces#control-service:~:text=Parameterized%20SQL-,Control%20Service,-Job%20Lifecycle%20API):** Essential for orchestrating the execution of Data Jobs.
+- **Jupyter instance:** Needed to access Jupyter Notebooks.
```

Comment:
if you install vdk-jupyterlab-extension it comes with jupyter instance so it's redundant requisite. REmove it.